### PR TITLE
Add an i3_to_json converter script

### PR DESCRIPTION
**Pull request by tianluyuan**
This is a rough script to read in i3 files and convert it into realtime's json format for use with the skymap scanner. Perhaps it could be pieced apart into the scanner itself at some point.

### CODE REVIEW

**Comment by ric-evans** [comment]
Could you make this a module in `skymap_scanner/utils`? This way we can give the scanner a `.i3` directly.

**Comment by ric-evans** [comment]
This has been requested several times, so let's fast-track it.

It will close https://github.com/icecube/skymap_scanner/issues/7, right @mlincett @tianluyuan?

**Comment by ric-evans** [comment]
@mlincett's comments in https://github.com/icecube/skymap_scanner/issues/32 are also relevant here

File: skymap_scanner/server/start_scan.py [hunk 1]
```diff
@@ -1013,18 +1013,30 @@ def _validate_arg(val: T, test: bool, exc: Exception) -> T:
         raise NotADirectoryError(args.gcd_dir)
 
     # read event file
-    if args.event_file.suffix == ".json":
-        # json
+    # .I3
+    if args.event_file.suffix == ".i3":
```
Comment (tianluyuan) [inline skymap_scanner/server/start_scan.py:1017]: We should probably allow the suffix to be ".i3" or ".i3.[gz/zst/bz2]".

**Review by ric-evans — APPROVED** [review]
Let's integrate this into the Skymap Scanner Service when it arrives. For now a script is good

**Pull request merged**